Gravity/Gravity.NUnit.Integration/RSAPI_IntegrationTest.cs

```diff
@@ -0,0 +1,395 @@
+                        Console.WriteLine("Application import Complete.");
+                    }
+                    else
+                    {
+                        Console.WriteLine($"Using existing application");
+                    }
+
+                   
+
+                    _client.APIOptions.WorkspaceID = _workspaceId;
+
+                }
+                catch (Exception ex)
+                {
+                    throw new Exception("Error encountered while creating new RSAPI Client and/or Service Proxy.", ex);
+                }
+                finally
+                {
+                    Console.WriteLine("Created RSAPI and Service Factory.....");
+                }
+
+                Console.WriteLine("Creating TestObject Helper.");
+                //1 retry setting because I want to know if it fails quickly while debugging, may bump up for production
+                _testObjectHelper = new Test.Helpers.TestObjectHelper(_servicesManager, _workspaceId, 1);
+                Console.WriteLine("TestObject Helper Created.");
+
+            }
+            catch (Exception ex)
+            {
+                throw new Exception("Error encountered in Test Setup.", ex);
+            }
+            finally
+            {
+                Console.WriteLine("Exit Test Fixture Setup.....");
+            }
+        }
+        #endregion
+
+        #region "Teardown"
+        [TestFixtureTearDown]
+        public void Execute_TestFixtureTeardown()
+        {
+            try
+            {
+                Console.WriteLine("Test Teardown START.....");
+
+                //Delete workspace
+                Console.WriteLine("Deleting workspace.....");
+                if (!_debug)
+                {
+                    DeleteWorkspace.DeleteTestWorkspace(_workspaceId, _servicesManager, ConfigurationHelper.ADMIN_USERNAME, ConfigurationHelper.DEFAULT_PASSWORD);
+                    Console.WriteLine("Workspace deleted.....");
+                }
+                else
+                {
+                    Console.WriteLine("Not deleteing workspace because tests are using existing");
+                }                
+            }
+            catch (Exception ex)
+            {
+                throw new Exception("Error encountered in Test Teardown.", ex);
+            }
+            finally
+            {
+                Console.WriteLine("Test Teardown END.....");
+                Console.WriteLine("Test END.....");
+            }
+        }
+        #endregion
+
+        #region "Tests"
+
+        [Test, Description("Verify Test Object is Created")]
+        public void Valid_Gravity_Object_Created()
+        {
+            Console.WriteLine(System.Reflection.MethodBase.GetCurrentMethod().Name + " Created");
+
+            try
+            {
+                //Arrange
+                Console.WriteLine("Starting Arrangement....");
```

Comment:
These messages are a good example of the sort of thing that is really not needed. At a minimum, the "concluded" ones are not, because that's implied by hte next "Starting" statement.

Reply:
Will adjust according to standard practices